src/main/resources/templates/contributor.html

```diff
@@ -353,6 +353,9 @@ <h4>Account ID: <span id="accountId"></span></h4>
                                     <button id="stripeDashboardButton" type="submit" class="btn btn-primary">
                                         Stripe Dashboard
                                     </button>
+                                    <button id="stripeDeleteButton" type="submit" class="btn btn-primary">
+                                        Delete
+                                    </button>
```

Comment:
@Lumi3011 Butonul asta trebuie sa fie in propriul lui ``<form></form>`` element, de aia nu merge cum trebuie. 

Apesi pe el si ti se deschide dashboard-ul de Stripe deoarece in momentul de fata se afla in ``<form>``-ul de stripe dashboard, care are deja listener-ul lui. 

In principiu orice buton care face submit trebuie intotdeauna sa se afle in propriul form. Chiar daca este un formular fara field-uri, el tot submit face, la un formular gol :D

Reply:
Ok, am inteles, nu am lucrat cu form-uri prea mult si nu stiam asta.